src/Symfony/Component/Console/Descriptor/TextDescriptor.php

```diff
@@ -201,19 +201,25 @@ protected function describeApplication(Application $application, array $options
             $commands = $description->getCommands();
 
             foreach ($description->getNamespaces() as $namespace) {
+                $namespace['commands'] = array_filter($namespace['commands'], function ($name) use ($commands) {
+                    return isset($commands[$name]);
+                });
+
+                if (empty($namespace['commands'])) {
```

Comment:
I think we use `0 === count(...)` rather than `empty` for checking that an array has no element (refs #19320, #19312)